test/backend.test.js

```diff
@@ -48,7 +48,8 @@ const vuexState = {
   pending: [], // contractIDs we've just published but haven't received back yet
   loggedIn: false, // false | { username: string, identityContractID: string }
   theme: 'blue',
-  fontSize: 1
+  fontSize: 1,
+  appLogsFilter: ['error', 'warn']
```

Comment:
add in `info`

Reply:
So, the UI should I also change "error, warn and info", instead of "error, warn and debug"?

Reply:
Don't understand the question. The UI should have checkboxes for all types of log messages. In other words: `Info`, `Log`, `Warn`, `Error`, and `Debug`

Reply:
don't forget to add `info` ^_^

Reply:
Actually it's irrelevant for these tests, but sure, added!

Reply:
oh you're right, i didn't realize I was looking at the tests. *doh!*